src/page-modules/contact/components/input/input.tsx

```diff
@@ -18,31 +21,63 @@ export const Input = ({
   name,
   checked,
   value,
+  description,
   onChange,
 }: InputProps) => {
   const { t } = useTranslation();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const displayDescriptionModal = description && isModalOpen; // Condition to avoid rendering of useEffect inside DescriptionModal before opened.
+
+  const handleModal = () => {
+    setIsModalOpen(!isModalOpen);
+  };
+
   return (
     <div
       className={andIf({
         [style.container]: true,
         [style.rowDisplay]: type === 'checkbox' || type === 'submit',
       })}
     >
-      <label>
-        <Typo.span textType="body__primary">{t(label)}</Typo.span>
-      </label>
+      <div className={style.label_container}>
+        <label>
+          <Typo.span textType="body__primary">{t(label)}</Typo.span>
+        </label>
+
+        {description && (
+          <button
```

Comment:
Any reason to not use the Button component here?

Reply:
No, will update👍